languages/es-ES.js

```diff
@@ -1,169 +1,169 @@
+				usersRemoved.length ? `**Usuarios eliminados**\n${util.codeBlock('', usersRemoved.join(', '))}` : '',
+				guildsAdded.length ? `**Servidores agregados**\n${util.codeBlock('', guildsAdded.join(', '))}` : '',
+				guildsRemoved.length ? `**Servidores eliminados**\n${util.codeBlock('', guildsRemoved.join(', '))}` : ''
+			].filter(val => val !== '').join('\n'),
+			COMMAND_EVAL_DESCRIPTION: 'Evalúa Javascript arbitrario. Reservado para el dueño.',
+			COMMAND_EVAL_EXTENDEDHELP: [
+				'El comando eval ejecuta el código tal y como está escrito, cualquier error será capturado.',
+				'También usa la herramienta "flags". Escribe --silent, --depth=number o --async para personalizar la salida.',
+				'El flag --silent silencia la salida.',
+				'El flag --depth acepta un número, por ejemplo, --depth=2, para personalizar la profundidad de util.inspect.',
+				'El flag --async rodea el código en un AsyncFunction en el cual puedes usar await, sin embargo, si necesitas saber el valor de algo, necesitarás la palabra clave return.',
+				'El flag --showHidden activará la opción showHidden de util.inspect.',
+				'Si la salida es demasiado larga, será enviada como un archivo, o impresa en consola si el bot no tiene el permiso ATTACH_FILES.'
+			].join('\n'),
+			COMMAND_EVAL_ERROR: (time, output, type) => `**Error**:${output}\n**Tipo**:${type}\n${time}`,
+			COMMAND_EVAL_OUTPUT: (time, output, type) => `**Salida**:${output}\n**Tipo**:${type}\n${time}`,
+			COMMAND_EVAL_SENDFILE: (time, type) => `La salida era demasiado larga... enviada como archivo.\n**Tipo**:${type}\n${time}`,
+			COMMAND_EVAL_SENDCONSOLE: (time, type) => `La salida era demasiado larga... imprimiendo el resultado en consola.\n**Tipo**:${type}\n${time}`,
+			COMMAD_UNLOAD: (type, name) => `✅ Desactivada la pieza tipo ${type}: ${name}`,
+			COMMAND_UNLOAD_DESCRIPTION: 'Desactiva una pieza de Klasa.',
+			COMMAND_UNLOAD: (type, name) => `✅ Desactivada la pieza tipo ${type}: ${name}`,
+			COMMAND_TRANSFER_ERROR: '❌ El archivo ha sido transferido o nunca existió.',
+			COMMAND_TRANSFER_SUCCESS: (type, name) => `✅ Transferida la pieza tipo ${type}: ${name} con éxito.`,
+			COMMAND_TRANSFER_FAILED: (type, name) => `La transferencia de la pieza tipo ${type}: ${name} al Cliente ha fallado. Por favor, revisa la consola.`,
+			COMMAND_RELOAD: (type, name) => `✅ Recargada la pieza tipo ${type}: ${name}`,
+			COMMAND_RELOAD_ALL: (type) => `✅ Recargadas todas las piezas tipo ${type}.`,
+			COMMAND_RELOAD_DESCRIPTION: 'Recarga una pieza de Klasa, o todas las piezas de una colección.',
+			COMMAND_REBOOT: 'Reiniciando...',
+			COMMAND_REBOOT_DESCRIPTION: 'Reinicia el bot.',
+			COMMAND_PING: '¿Ping?',
+			COMMAND_PING_DESCRIPTION: 'Ejecuta una prueba de conexión a Discord.',
+			COMMAND_PINGPONG: (diff, ping) => `¡Pong! (El viaje duró: ${diff}ms. Latencia: ${ping}ms.)`,
+			COMMAND_INVITE_SELFBOT: '¿Para qué quiere un enlace de invitación un selfbot?',

```

Comment:
This line was removed in en-US or should be removed soon otherwise. We don't support selfbots so there's no point on supporting this line.

```suggestion
```